README.md

```diff
@@ -39,8 +39,10 @@
      | ResetOnDisconnect	|Determines if sequence numbers should be reset to 1 after an abnormal termination.	| Y / N | N |
      | ResetOnError	| Session setting for doing an automatic reset when an error occurs. A reset means disconnect, sequence numbers reset, store cleaned and reconnect, as for a daily reset.	| Y / N |	N |
      | DisconnectOnError	| Session setting for doing an automatic disconnect when an error occurs.	| Y / N	| N |    
-     |SessionAlias| session alias for incoming/outgoing th2 messages. | case-sensitive alpha-numeric string | |
-	
+     | SessionAlias| session alias for incoming/outgoing th2 messages. | case-sensitive alpha-numeric string | |
+     | QueueCapacity | Quantity of creating threads for sessions | integer value | |
```

Comment:
It's not an amount of threads - it's max message queue size in message dispatching thread